versions.txt

```diff
@@ -2,20 +2,20 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
-opentelemetry-collector=0.93.0
+opentelemetry-collector=0.94.0
```

Comment:
i think the collector is at 0.94.1 https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.94.1

Reply:
The libraries are, but not the released images: https://github.com/open-telemetry/opentelemetry-collector-releases/releases